Sources/TuistLoader/Utils/ManifestFilesLocator.swift

```diff
@@ -101,9 +101,21 @@ public final class ManifestFilesLocator: ManifestFilesLocating {
             Manifest.workspace.fileName(path),
             Manifest.plugin.fileName(path),
         ]
-        let tuistManifestsFilePaths = fileHandler.glob(path, glob: "**/*.swift")
-            .filter { fileNamesCandidates.contains($0.basename) }
-            .filter { hasValidManifestContent($0) }
+
+        var tuistManifestsFilePaths = [AbsolutePath]()
+
+        let enumerator = FileManager.default.enumerator(atPath: path.pathString)
+        while let candidatePath = enumerator?.nextObject() as? String {
+            let candidateURL = URL(fileURLWithPath: candidatePath)
+            if fileNamesCandidates.contains(candidateURL.lastPathComponent) {
+                let path = candidateURL.absoluteString.replacingOccurrences(of: "file://", with: "")
+                let absolutePath = AbsolutePath(stringLiteral: path)
```

Comment:
Doesn't URL have a `path` property?
```suggestion
                let absolutePath = AbsolutePath(stringLiteral: candidateURL.path)
```